packages/devtools_shared/pubspec.yaml

```diff
@@ -17,3 +18,5 @@ dependencies:
 
 dev_dependencies:
   test: ^1.21.2
+  build_runner: ^2.0.4
+  mockito: ^5.1.0
```

Comment:
do we still need all these changes to the pubspec? they don't look related to any devtools_shared changes in this PR